docs/releasing.rst

```diff
@@ -0,0 +1,34 @@
+:orphan:
+
+Releasing
+=========
+
+This is the release process for the client itself.
+
+* First update the version in ``package.json`` **and** ``index.js``.
+  The versions use `semantic versioning <https://semver.org/>`_.
+* Run ``npm install && npm run build``.
+  This ensures you have the latest dependencies and you've built
+  the latest version of the documentation.
+* Run ``npm run changelog`` and update ``CHANGELOG.rst``
+  (included by :doc:`/changelog`)
+  with the release date and details.
+* Commit these changes, create a pull request, and merge it.
+* Tag the release:
+  
+    .. code-block:: bash
+
+        export VERSION=vX.Y.Z
```

Comment:
Seems unnecessary if we're only using it once in the script, but I guess it makes the version more explicit.

Reply:
Having configurable values not buried in the middle results in fewer copy/paste errors in my experience.